src/infrastructure/figma/figma-service.ts

```diff
@@ -204,17 +204,22 @@ export class FigmaService {
 			);
 
 			if (!devResourceToDelete) {
-				getLogger().error(
+				getLogger().info(
 					`No matching dev resource found to delete for file ${designId.fileKey} and node ${designId.nodeId}`,
 				);
 				return;
 			}
 
-			await figmaClient.deleteDevResource({
-				fileKey: designId.fileKey,
-				devResourceId: devResourceToDelete.id,
-				accessToken,
-			});
+			try {
+				await figmaClient.deleteDevResource({
+					fileKey: designId.fileKey,
+					devResourceId: devResourceToDelete.id,
+					accessToken,
+				});
+			} catch (e) {
+				if (e instanceof NotFoundHttpClientError) return;
```

Comment:
**note:** Ignore an error indicating that a _Dev Resource_ does not exist anymore (e.g., it has been deleted before).